gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
Discussed offline and understood pros and cons, but we are still not convinced by each other, but as this is a small issue and I'd like to follow what other ruby projects are doing, we'll just keep it as is now and we are both fine with it.